tensorflow/python/distribute/BUILD

```diff
@@ -622,6 +622,7 @@ distribute_py_test(
     shard_count = 10,
     tags = [
         "multi_and_single_gpu",
+        "no_rocm", #MultiMirroredWorker2x 
```

Comment:
What does `MultiMirroredWorker2x` mean?  Maybe instead leave a reference to a github issue?

Reply:
That's a reference to what's causing the issue. I'll remove the comment all-together, because we have an issue for it on our forked repo, and it'll get addressed there then pushed upstream when it is set.